config/locales/views/layouts/application/no.yml

```diff
@@ -11,8 +11,9 @@
         login: "Logg inn"
         logout: "Logg ut"
         change_language: "English"
-        membership: "Medlemskort"
+        membership: "Registrer/endre medlemskap"
```

Comment:
Just a minor nibble with this: It makes the menu item have two lines of text (instead of one), so I would probably prefer only `Membership/Medlemskap`. Otherwise the PR looks good.